fairscale/nn/wrap/auto_wrap.py

```diff
@@ -4,23 +4,72 @@
 # LICENSE file in the root directory of this source tree.
 
 import contextlib
-from typing import Any, Callable, Dict, Generator, List, Optional, Tuple
+from typing import Any, Callable, Dict, Generator, Optional, Set, Tuple, cast
 
 import torch.nn as nn
 
-from fairscale.nn.data_parallel.fully_sharded_data_parallel import FullyShardedDataParallel
 from fairscale.nn.misc import checkpoint_wrapper
 
-# Modules that don't wrap.
-FSDP_MODULE_EXCLUDE_WRAP = {nn.ModuleList, nn.ModuleDict}
-# Modules that we don't recurse down to their children.
-FSDP_MODULE_BLOCKLIST = {nn.MultiheadAttention}
+
+def should_wrap_default(
+    should_recurse: bool,
+    module: nn.Module,
+    unwrapped_params: int,
+    # These are customizable for this default policy function.
+    min_num_params: int = int(1e8),
+    blacklist: Optional[Set] = None,
+    exclude_wrap: Optional[Set] = None,
+) -> bool:
+    """Default policy function on with :func:`auto_wrap`.
```

Comment:
nit: replace "function on with ..." -> "function for ..."

Reply:
done. I must have been debating between "on" and "with". Lol.